src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lambda-telemetry.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Amazon Lambda Telemetry integration
+tags:
+  - Integrations
+  - Amazon integrations
+  - AWS integrations list
+metaDescription: "New Relic's Amazon App Runner monitoring integration: what data it reports, and how to enable it."
```

Comment:
why App Runner? Lambda telemetry extension is another project, nothing common with App Runner